charts/shared/common-res/values-test.yaml

```diff
@@ -85,6 +85,146 @@ ClusterRoleBinding:
     _:
       enabled: true
 
+Deployment:
+  enabled: true
+  items:
+    test-dict:
+      podSpec:
+        spec:
+          securityContext:
+            runAsUser: 1000
+            runAsGroup: 3000
+            fsGroup: 2000
+          initContainers:
+            test:
+              command: ['/bin/echo', 'init']
+          containers:
+            test:
+              command: ['/bin/echo', 'run']
+    test-list:
+      podSpec:
```

Comment:
Need a discussion on this

Reply:
Discussion took place and PR updated accordingly.